frontend/javascripts/admin/tasktype/task_type_create_view.js

```diff
@@ -290,27 +290,29 @@ class TaskTypeCreateView extends React.PureComponent<Props, State> {
                 prevValues.tracingType !== curValues.tracingType
               }
             >
-              {({ getFieldValue }) =>
-                getFieldValue(["tracingType"]) !== "volume" ? (
-                  <React.Fragment>
-                    <FormItem
-                      name={["settings", "somaClickingAllowed"]}
-                      label="Settings"
-                      valuePropName="checked"
-                    >
-                      <Checkbox>Allow Single-node-tree mode (&quot;Soma clicking&quot;)</Checkbox>
-                    </FormItem>
-
-                    <FormItem name={["settings", "branchPointsAllowed"]} valuePropName="checked">
-                      <Checkbox>Allow Branchpoints</Checkbox>
-                    </FormItem>
-
-                    <FormItem name={["settings", "mergerMode"]} valuePropName="checked">
-                      <Checkbox>Merger Mode</Checkbox>
-                    </FormItem>
-                  </React.Fragment>
-                ) : null
-              }
+              {({ getFieldValue }) => (
+                <div
+                  style={{
```

Comment:
I'd propose to add a comment so that this doesn't happen again in the future

Reply:
Very good point :+1: